packages/board-server/src/server/types.ts

```diff
@@ -111,8 +112,22 @@ export type RunBoardStateStore = {
     ticket: string
   ): Promise<ReanimationState | undefined>;
   saveReanimationState(user: string, state: ReanimationState): Promise<string>;
+  getBoardServerCorsConfig(): Promise<BoardServerCorsConfig | undefined>
```

Comment:
I would do something different here. The `RunBoardStateStore` is a subset of the larger `BoardServerStore` type that needs to be defined. Then, `BoardServerStore` extends it. The `RunBoardStateStore` is specifically about storing the reanimation state. But this is something that can be done as a follow-up patch.

Reply:
Yeah, I agree - I actually started down this path, having a `BoardServerStore` and `RunBoardStateStore` but I ran into difficulties with union types and then forgot to address it again. I'll have a play with it today and see if I can get it to work, but if not, I'll do this in a follow up.